test/e2e/basic_publish.test.ts

```diff
@@ -110,6 +110,22 @@ describe("publish a message", () => {
 
       await eventually(async () => expect((await rabbit.getQueueInfo(stream)).messages).eql(howMany * 2), 10000)
     }).timeout(30000)
+
+    it("is not active if create a publisher with null publisherRef", async () => {
```

Comment:
```suggestion
    it("is not active if create a publisher without publisherRef", async () => {
```

Ti suggerirei di rinominare lo stesso con lo stesso nome in "is not active if create a publisher with empty publisherRef”